### PR TITLE
Assault Ops update and fixes

### PR DESCRIPTION
**Pull request by Cenrus**
## About The Pull Request

I've had a look at assault operatives and changed some things to hopefully make it a better experience.

I've removed some armament selections such as energy shields and NVGs as these are already provided for free, so there's no reason to buy them. I've also added a selection of 3 modsuit modules to the armament dispenser.

Fixed the issue where purchased guns didn't come with any ammo box with spare magazines. Removed leg cuffs from the cruiser because they don't actually do anything.

Added syndicate prisoner outfits and added them to the shuttle, these dark red suits don't have suit sensors to prevent easy tracking of the syndicate ship. I also added them to the DS2 inmate spawner. 

## How This Contributes To The Skyrat Roleplay Experience

I like the idea of the mode, but it needed some polishing. I didn't want to see some poor sap spend his points on an item that they don't have to pay for, anyway.

## Changelog

:cl:
add: Assault ops can now buy MODsuit modules at their vendor
fix: Fixed assault ops not dispensing extra ammo for their weapons
del: Removed some useless armament entries
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Add newlines and we're gucci

File: modular_skyrat/modules/assault_operatives/code/armaments/__armament_bodyarmor.dm [hunk 1]
```diff
@@ -28,9 +28,6 @@
...
 
 /datum/armament_entry/assault_operatives/bodyarmor/elite_modsuit
+	name = "Elite Syndicate MODsuit"
 	item_type = /obj/item/mod/control/pre_equipped/elite
-	cost = 15
-
-/datum/armament_entry/assault_operatives/bodyarmor/elite_modsuit_flamer
-	item_type = /obj/item/mod/control/pre_equipped/elite/flamethrower
-	cost = 18
+	cost = 15
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/assault_operatives/code/armaments/__armament_bodyarmor.dm]:
```suggestion
	cost = 15

```

File: modular_skyrat/modules/assault_operatives/code/armaments/armament_modules.dm [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	cost = 3
+	
+/datum/armament_entry/assault_operatives/modules/flamethrower
+	item_type = /obj/item/mod/module/flamethrower
+	cost = 3
+	
+/datum/armament_entry/assault_operatives/modules/energy_shield
+	description = "Projects three self recharging forcefields around the suit, protecting the user from incoming attacks."
+	item_type = /obj/item/mod/module/energy_shield
+	cost = 5
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/assault_operatives/code/armaments/armament_modules.dm]:
```suggestion
	cost = 5

```

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 3]
```diff
@@ -143,3 +143,9 @@
 	greyscale_config_inhand_left = null
 	greyscale_config_inhand_right = null
 	greyscale_config_worn = null
+
+/obj/item/clothing/under/rank/prisoner/syndicate
+	name = "syndicate prisoner jumpsuit"
+	desc = "A crimson red jumpsuit worn by syndicate captives. Its sensors have been shorted out."
+	greyscale_colors = "#992300"
+	has_sensor = FALSE
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm]:
```suggestion
	has_sensor = FALSE

```

**Comment by Cenrus** [comment]
Went out right after I published the PR, should be fine now

**Pull request merged**